### PR TITLE
Fixed: comments are no longer removed

### PR DESCRIPTION
**Pull request by uniquexor**
Used to remove everything between a semicolon and a nonempty token. Now it jumps over the comments leaving them as is only removing whitespaces between a comment and a semicolon. For exmple, if we have code like this:
`$test = $a /* + $b */  ;`
Before the comment would be removed:
`$test = $a;`
Now the comment remains:
`$test = $a; /* + $b */`

### CODE REVIEW

**Comment by aik099** [comment]
Can you maybe rephrase so that PR has:

- short title that explains what is done
- description that explains in detail if needed

Maybe add some examples describing what is changed.

**Comment by uniquexor** [comment]
Sorry. Have updated the name and description.

**Comment by aik099** [comment]
It obviously changes the placement of the comment itself (was before `;` but now it's after `;`). Is that acceptable?

**Comment by uniquexor** [comment]
I think it is more acceptable than just removing it entirely, no?

**Comment by aik099** [comment]
Yes, but people who were expecting old fixing behavior might be surprised to see how it will work after a fix. I personally don't place `// ...` comments on same line with a code. Just a guessing here really.

**Comment by uniquexor** [comment]
Neither do I, but I have colleagues, who do :) I can imagine a situation where some code has been commented out and after running a beautifier it would be lost. For example, imagine a situation like this:
`$c = $b /* + $c */ ;`
If someone, for some reason temporarily commented out the code, it would get lost after running this sniff.
In my opinion running a code beautifier, should not result in a loss of code, even if it's commented out.

**Comment by aik099** [comment]
That's more close to life example. You'd better also mention `/* ... */` in your unit tests to confirm, that it isn't removed.

**Comment by uniquexor** [comment]
What do you mean? There is a case for `/* ... */` comment in both JS and PHP unit tests.

**Comment by aik099** [comment]
Ah, then I must have missed that. Sorry. Then all is fine.

**Comment by aik099** [comment]
I've meant exactly what you've shown, in tests:

```php
$c = $b /* + $c */ ;
```

There are only examples with comment coming after `;` before fixing.

**Comment by uniquexor** [comment]
Well, there is a line in PHP test:
`$test = $this->testThis()  /* comment here */  ;`
and in JS test:
```
alert('hi')   /* Comment between */
;
```
Both, in my opinion, cover your case

**Comment by uniquexor** [comment]
Sorry, I'm new to git... The last commit should have probably been a new pull request. Is there a way for me to remove it from here?

**Comment by jrfnl** [comment]
@uniquexor You can hard reset your branch to the previous commit. This will cause you to lose the change though. If you're working on several issues, it might be useful to start working in separate branches for each.
